test/cluster/functionality.sh

```diff
@@ -108,6 +108,21 @@ for elasticsearch_pod in $( oc get pods --selector component="${OAL_ELASTICSEARC
 	for template in $( oc exec "${elasticsearch_pod}" -- ls -1 /usr/share/java/elasticsearch/index_templates ); do
 		os::cmd::expect_success_and_text "curl_es '${elasticsearch_pod}' '/_template/${template}' -X HEAD -w '%{response_code}'" '200'
 	done
+
+	os::log::info "Checking that Elasticsearch pod ${elasticsearch_pod} has expected plugins installed"
+	matching_plugins=0
+	for plugin in $( curl_es "${elasticsearch_pod}" '/_cat/plugins?local=true&h=component' ); do
+	    if [ "${plugin}" == "search-guard2" ]; then
```

Comment:
This I believe will always fail now that SG is included as part of our plugin

Reply:
IIRC it is still listed in this _cat output

Reply:
when we move to es5, will this fail because it will report `search-guard5` or just `search-guard`?  That is, can we "future proof" this by looking for just "search-guard" in the pattern?

Reply:
It will say [`search-guard-5`](https://github.com/floragunncom/search-guard/blob/5.5.0/plugin-descriptor.properties#L9). So may be we can do something like?:

````bash
if [[ "${plugin}" == "search-guard"* ]];
````

Reply:
But in that case we probably want to distinguish between plugins `search-guard-5.x` and [`search-guard-ssl`](https://github.com/floragunncom/search-guard-ssl/blob/master/plugin-descriptor.properties#L9) 'cos I think both plugins will be present.

Reply:
ok - then add a case for `search-guard-ssl` before the case for the general `plugin contains 'search-guard'`

Reply:
I believe we package the ssl plugin as well.  I would expect a check for it to fail too.